
---
float64 dist
float64 av_speed_x
float64 av_speed_y